<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리OTF Black</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Break Captcha - v1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Developed by 아주대학교 - 사이버보안학과 (이상민, 김형호, 송연주, 조영준)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Powered by PyQt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>프로젝트 전체 진행 관리</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>딥러닝 알고리즘 학습 및 테스트</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>문자 인식 라이브러리, tesseract 테스트</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이상민</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Selenium을 통한 사용자 정보 자동입력</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데이터 수집 자동화</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>음성 STT, CSR, API 테스트</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>이미지 전처리 과정 구현</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>머신러닝 알고리즘 학습 및 적용</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/robot.ico</normaloff>:/newPrefix/robot.ico</iconset>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GUI 설계 및 골격 제작</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>음성 STT, 구글 speech 테스트</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>음성 전처리 과정 구현</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="listWidget_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GUI 상세 기능 구현</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>사용자 정보 저장 기능 구현</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>데이터 Train, Test 분류</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>김형호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>송연주</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>조영준</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>80</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>261</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgb(0, 0, 255);
</string>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="linkButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>261</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>경기천년제목 Light</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 150);
border-color: rgb(0, 0, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&gt; Github</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>robot.ico</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
